src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -321,6 +324,71 @@ def fuzz(self,
                                 artifact_prefix, additional_args, extra_env)
 
 
+class FuchsiaQemuLibFuzzerRunner(new_process.ProcessRunner, LibFuzzerCommon):
+  """libFuzzer runner (when Fuchsia is the target platform)."""
+
+  SSH_RETRIES = 10
+  SSH_WAIT = 2
+
+  def __init__(self, executable_path, default_args=None):
+    if (not environment.get_value('FUCHSIA_PKEY_PATH')) or (
```

Comment:
environment.get_value('FUCHSIA_PKEY_PATH') and FUCHSIA_PORTNUM in a local var and then use in this if and usage below.